openfisca_core/tracers.py

```diff
@@ -99,9 +120,19 @@ def trees(self):
     def computation_log(self):
         return ComputationLog(self)
 
+    @property
+    def performance_log(self):
+        return PerformanceLog(self)
+
+    def _get_time_in_sec(self) -> float:
+        return time.time_ns() / (10**9)
```

Comment:
:exclamation: This removes the 3.6 backward compatibility

New in version 3.7.
From https://docs.python.org/3/library/time.html#time.time_ns